tests/test_entities.py

```diff
@@ -1313,6 +1316,7 @@ def setUpClass(cls):
                 entities.Organization(**generic).download_debug_certificate,
                 'get'
             ),
+            (entities.Host(**generic).add_puppetclass, 'post'),
```

Comment:
`GenericTestCase` is designed for methods that don't require any extra parameters. For full coverage, a new test case would need to be written.

> Maybe it requires separate test but it would be full copy of test_generic with single line changed :(

This test method would need slightly more than one line of changes. The `self.assertEqual(client_request.call_args[1], kwargs)` assertion would need to be modified or dropped, as would the line that sets `kwargs`.